src/bda/plone/productshop/browser/configure.zcml

```diff
@@ -174,7 +174,6 @@
   <!-- forms -->
   <adapter
     for="Products.CMFCore.interfaces.IFolderish
-         Products.CMFDefault.interfaces.ICMFDefaultSkin
```

Comment:
This is the add form, which is always expected as a multid adapter between context, request and FTI.

afaict the correct fix would be two replace ``Products.CMFDefault.interfaces.ICMFDefaultSkin`` by ``zope.publisher.interfaces.browser.IDefaultBrowserLayer``. 

Just removal will break it.

Getting rid of ``Products.CMFDefault`` is important, it was removed from Plone 5.1 already as core-dependency.